projects/experimental/directives/appearance/appearance.directive.ts

```diff
@@ -0,0 +1,22 @@
+import {Directive, Input} from '@angular/core';
+import {TuiInteractiveStateT} from '@taiga-ui/core';
+
+@Directive({
+    selector: '[tuiAppearance]',
+    host: {
+        tuiAppearance: '',
```

Comment:
isn't this a duplicate selector? (same in tuiSurface)

Reply:
We need it to make sure we target appearance directive and not some other data-appearance. If you write `[tuiAppearance]="variable"` it does not add `tuiAppearance` attribute.